### PR TITLE
Disable Wyre

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

 none

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

### CODE REVIEW

File: src/controllers/action-queue/runtime/evaluateAction.ts [hunk 1]
```diff
@@ -145,61 +142,27 @@ export async function evaluateAction(context: ExecutionContext, program: ActionP
...
-      const parentCurrencyCode = wallet.currencyInfo.currencyCode
-      const currencyCode = getCurrencyCode(wallet, tokenId)
-
-      const wyreClient = await makeWyreClient({
-        account
-      })
+    // TODO: Remove once we implement action-queue disk data deletion
+    case 'wyre-buy': {
+      const makeExecutionOutput = async (): Promise<ExecutionOutput> => {
+        console.error(new Error(`Using obsolete wyre-buy action`))
```
Comment (swansontec) [inline src/controllers/action-queue/runtime/evaluateAction.ts:148]: The `new Error` shouldn't be needed. You can print strings to the console.

File: src/controllers/action-queue/runtime/evaluateAction.ts [hunk 2]
```diff
@@ -145,61 +142,27 @@ export async function evaluateAction(context: ExecutionContext, program: ActionP
...
+          broadcastTxs: []
         }
+      }
+
+      return makeExecutableAction(context, makeExecutionOutput)
+    }
+    // TODO: Remove once we implement action-queue disk data deletion
+    case 'wyre-sell': {
+      const makeExecutionOutput = async (): Promise<ExecutionOutput> => {
+        console.error(new Error(`Using obsolete wyre-sell action`))
```
Comment (swansontec) [inline src/controllers/action-queue/runtime/evaluateAction.ts:161]: Same for `new Error`

File: src/controllers/action-queue/runtime/evaluateAction.ts (hunk 1 above)
Comment (samholmes) [reply src/controllers/action-queue/runtime/evaluateAction.ts]: Fixed

File: src/controllers/action-queue/runtime/evaluateAction.ts (hunk 2 above)
Comment (samholmes) [reply src/controllers/action-queue/runtime/evaluateAction.ts]: Fixed

**Review by samholmes — APPROVED** [review]
William and I agree this is good to go.

**Pull request merged**